error: 'Path' is not a member type of 'Path'

I'm not sure if this is my issue, but I get this error when I try to do anything with Path on Google Colab:

```
expression produced error: error: /tmp/expr56-6421f7..swift:1:70: error: 'Path' is not a member type of 'Path'
Swift._DebuggerSupport.stringForPrintObject(Swift.UnsafePointer<Path.Path>(bitPattern: 0x7f6d9bca1340)!.pointee)
                                                                ~~~~ ^

Path.Path:1:36: note: 'Path' declared here
@dynamicMemberLookup public struct Path : Equatable, Hashable, Comparable {
```

My code:
```swift
%install-location $cwd/swift-install
%install '.package(url: "https://github.com/mxcl/Path.swift", from: "0.16.1")' Path
```

```swift
import Path
Path("/")
// or:
Path.root
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -6,11 +6,13 @@ on:
       - .github/workflows/ci.yml
   schedule:
     - cron: '3 3 * * 5'  # 3:03 AM, every Friday
+concurrency:
+  group: ${{ github.head_ref || 'push' }}
+  cancel-in-progress: true
 jobs:
   smoke:
     runs-on: macos-latest
     steps:
-    - uses: technote-space/auto-cancel-redundant-job@v1
     - uses: actions/checkout@v2
     - run: swift test --generate-linuxmain
     - run: git diff --exit-code

--- a/Sources/Path+CommonDirectories.swift
+++ b/Sources/Path+CommonDirectories.swift
@@ -108,3 +108,13 @@ func defaultUrl(for searchPath: FileManager.SearchPathDirectory) -> DynamicPath 
 }
 #endif
 
+#if swift(>=5.5)
+extension Pathish where Self == Path {
+    static var home: DynamicPath { Path.home }
+    static var root: DynamicPath { Path.root }
+    static var cwd: DynamicPath { Path.cwd }
+    static var documents: DynamicPath { Path.documents }
+    static var caches: DynamicPath { Path.caches }
+    static var applicationSupport: DynamicPath { Path.applicationSupport }
+}
+#endif

--- a/Sources/Path.swift
+++ b/Sources/Path.swift
@@ -7,6 +7,8 @@ import func Glibc.realpath
 let _realpath = Glibc.realpath
 #endif
 
+public typealias PathStruct = Path
+
 /**
  A `Path` represents an absolute path on a filesystem.
 

--- a/Tests/PathTests/PathTests.swift
+++ b/Tests/PathTests/PathTests.swift
@@ -3,7 +3,20 @@ import func XCTest.XCTAssertEqual
 import Foundation
 import XCTest
 
+extension PathStruct {
+    var foo: Int { fatalError()}
+}
+
 class PathTests: XCTestCase {
+    func testNewStuff() {
+    #if swift(>=5.5)
+        func foo<P: Pathish>(_ path: P) {}
+
+        foo(.home)
+        foo(.root)
+    #endif
+    }
+
     func testConcatenation() {
         XCTAssertEqual((Path.root/"bar").string, "/bar")
         XCTAssertEqual(Path.cwd.string, FileManager.default.currentDirectoryPath)

--- a/Tests/PathTests/XCTestManifests.swift
+++ b/Tests/PathTests/XCTestManifests.swift
@@ -45,6 +45,7 @@ extension PathTests {
         ("testMktemp", testMktemp),
         ("testMoveInto", testMoveInto),
         ("testMoveTo", testMoveTo),
+        ("testNewStuff", testNewStuff),
         ("testNoUndesiredSymlinkResolution", testNoUndesiredSymlinkResolution),
         ("testOptionalInitializer", testOptionalInitializer),
         ("testParent", testParent),

